switchapi/switch_nhop_int.h

```diff
@@ -76,8 +76,8 @@ extern "C" {
 
 #define switch_nhop_get_group(_device, _handle, _info)                    \
   ({                                                                \
-    switch_nhop_info_t *_tmp_nhop_info = NULL;                      \
-    (void)(_tmp_nhop_info == *_info);                               \
+    switch_nhop_group_info_t *_tmp_nhop_group_info = NULL;          \
+    (void)(_tmp_nhop_group_info == *_info);                         \
```

Comment:
I don't understand. `(_tmp_nhop_group_info == *_info)` is a Boolean expression. Why are we casting a `bool` to `void`?

Reply:
May I suggest the following?
```
#define switch_nhop_get_group(_device, _handle, _info)                    \
    switch_handle_get(                                              \
        _device, SWITCH_HANDLE_TYPE_NHOP_GROUP, _handle, (void **)(_info))
```